README.md

```diff
@@ -159,13 +174,75 @@ To uninstall/delete a deployment with name `my-splunk-otel-collector`:
 $ helm delete my-splunk-otel-collector
 ```
 
-The command removes all the Kubernetes components associated with the chart and
-deletes the release.
+## Log Architecture
 
-## Advanced Configuration
+Splunk OpenTelemetry Collector for Kubernetes deploys a DaemonSet on each node. And in the DaemonSet, a OpenTelemetry container runs and does the collecting job. Splunk OpenTelemetry Collector for Kubernetes uses the [node logging agent](https://kubernetes.io/docs/concepts/cluster-administration/logging/#using-a-node-logging-agent) method. See the [Kubernetes Logging Architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/) for an overview of the types of Kubernetes logs from which you may wish to collect data as well as information on how to set up those logs.
```

Comment:
Please move this to docs folder and provide link here. The README should have basic getting started information and product docs or docs folder should have additional information.